### DISCUSSION TITLE
Support physical light transport for simulating interference and diffraction (wave optics)

### DISCUSSION THREAD

**Discussion by ghost** [mitsuba-renderer/mitsuba3#1031] [Feature requests]
There are some attempts trying to mimic wave optics in a ray tracer. 

- [A Generalized Ray Formulation For Wave-Optics Rendering](https://ssteinberg.xyz/2023/03/27/rtplt/)
- [Towards Practical Physical-Optics Rendering](https://ssteinberg.xyz/2022/04/03/practical_plt/)
- [Physical Light-Matter Interaction in Hermite-Gauss Space](https://ssteinberg.xyz/2021/07/31/physical_light_matter_interaction_hg_space/)
- [A Generic Framework for Physical Light Transport](https://ssteinberg.xyz/2021/04/26/generic_physical_light_transport_framework/)

Is it possible to do similar things in mitsuba?

**Comment by njroussel** [comment]
Hi @nick008a 

AFAIK none of these techniques are implemented in Mitsuba. Implementing them most likely requires a lot of work, this is not something we are pursuing currently.
